avogadro/io/cjsonformat.cpp

```diff
@@ -152,6 +152,14 @@ bool CjsonFormat::read(std::istream& file, Molecule& molecule)
     }
   }
 
+  // formal charges
+  json charges = atoms["charges"];
```

Comment:
Sorry, I should have been explicit. Let's call it "formalCharges" because we'll probably want "partialCharges" sometime.